fr/errors.po

```diff
@@ -0,0 +1,392 @@
+msgid ""
+msgstr ""
+"Content-Type: text/plain; charset=utf-8\n"
+"Project-Id-Version: \n"
+"POT-Creation-Date: \n"
+"PO-Revision-Date: \n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Language: fr\n"
+"X-Generator: Poedit 2.0.7\n"
+
+msgid "**Embed Links** permission is required, please fix or tell an admin."
+msgstr ""
+"La permission **Intégrer des liens** est requise, merci de régler le problème ou "
+"de contacter un administrateur."
+
+msgid "Interactions via Direct/Private messages with bot have been **disabled**."
+msgstr ""
+"Les interactions via les messages privés pour ce bot ont été **désactivées**."
+
+msgid "Missing Arguments"
+msgstr "Arguments manquants"
+
+msgid "Missing valid languages and text for translation, please edit to fix."
+msgstr ""
+"Langue valide et texte à traduire manquants, merci d'éditer pour corriger le "
+"problème."
+
+msgid "Missing Text"
+msgstr "Texte manquant"
+
+msgid "Please provide some text for language detection."
+msgstr "Merci d'insérer du texte pour détecter la langue."
+
+msgid ""
+"To translate a message, you *need* a message!\n"
+"Please edit your command to include text."
+msgstr ""
+"Pour traduire un message, vous *avez besoin* d'un message !\n"
+"Merci d'éditer votre commande pour y inclure du texte."
+
+msgid "Invalid languages argument"
+msgstr "Arguments de langue invalides"
+
+msgid "Incorrect format for from/to argument, please edit to fix."
+msgstr ""
+"Format incorrect de l'argument from/to, merci d'éditer pour corriger le problème."
```

Comment:
from/to not translated?

Reply:
Hmm we thought this was an command argument, can we have some context?